msgpack-core/src/main/java/org/msgpack/core/MessagePack.java

```diff
@@ -38,6 +38,12 @@
     public static final Charset UTF8 = Charset.forName("UTF-8");
 
     /**
+     * Default packer/unpacker configurations
+     */
+    public static final PackerConfig DEFAULT_PACKER_CONFIG = new PackerConfig();
+    public static final UnpackerConfig DEFAULT_UNPACKER_CONFIG = new UnpackerConfig();
```

Comment:
Make these variable scopes to package by removing `public` since these are only for internal use.